src/test/scala/org/broadinstitute/dsde/rawls/webservice/EntityApiServiceSpec.scala

```diff
@@ -800,15 +805,48 @@ class EntityApiServiceSpec extends ApiServiceSpec {
         assertResult(StatusCodes.OK) {
           status
         }
+
+        def attrToString(a: Attribute): String = {
+          a match {
+            case AttributeString(value) => value
+            case AttributeNumber(value) => value.toString
+            case AttributeValueList(list) if list.isEmpty => ""
+            case AttributeValueList(list) => attrToString(list.head)
+            case _ => throw new RawlsException("make the compiler stop whining")
+          }
+        }
+
         assertResult(EntityQueryResponse(
           defaultQuery.copy(sortField = "mixed", pageSize = paginationTestData.numEntities),
           EntityQueryResultMetadata(paginationTestData.numEntities, paginationTestData.numEntities, 1),
-          paginationTestData.entities.sortBy(_.attributes("mixed") match {
-            case AttributeString(value) => value
-            case AttributeNumber(value) => value.toString
+          paginationTestData.entities.sortBy( e => attrToString(e.attributes("mixed")) ))) {
+          responseAs[EntityQueryResponse]
+        }
+      }
+  }
+
+  it should "return sorted results on entity query for mixed numeric-type field including lists" in withPaginationTestDataApiServices { services =>
+    Get(s"/workspaces/${paginationTestData.workspace.namespace}/${paginationTestData.workspace.name}/entityQuery/${paginationTestData.entityType}?sortField=mixedNumeric&pageSize=${paginationTestData.numEntities}") ~>
+      sealRoute(services.entityRoutes) ~>
+      check {
+        assertResult(StatusCodes.OK) {
+          status
+        }
+
+        def attrToNumeric(a: Attribute): BigDecimal = {
+          a match {
+            case AttributeNumber(value) => value
+            // this offset magic with paginationTestData.numEntities*2 guarantees that lists "are bigger than"
+            // numbers without reimplementing NumericAttributeOrderingAsc in the test
+            case AttributeValueList(list) => list.size + (paginationTestData.numEntities*2)
```

Comment:
After thinking about it for a while, this makes sense. If I understand this correctly, any number greater than `numEntities` should do the trick.